index.js

```diff
@@ -33,6 +33,9 @@ function S3Storage (opts) {
   if (!opts.accessKeyId) throw new Error('accessKeyId is required')
   if (!opts.region) throw new Error('region is required')
   if (!opts.dirname) throw new Error('dirname is required')
+  if (!opts.acl) console.log('--------------------------------------------------------------\n'+
+                              '[MULTER S3] ACL is not specified, will used private by default\n'+
+                              '--------------------------------------------------------------')
```

Comment:
@LinusU  Okay, removed
My thought was to warn that the default *acl * was private, but yeah that can be notified in the doc